README.md

```diff
@@ -25,9 +27,8 @@ import { Client } from "@microsoft/microsoft-graph-client";
 import 'isomorphic-fetch';
 
 const client = Client.initWithMiddleware({
-	debugLogging: true,
-	middleware,
 	defaultVersion: 'beta',
+	...
```

Comment:
Is this an intended change?

Reply:
Yes. Suggestion by @MIchaelMainer

Reply:
ok, just wanted to make sure because I couldn't relate it to the PR title.